app/Http/Controllers/EquipmentTypeController.php

```diff
@@ -16,9 +16,9 @@ public function index()
     {
         //
         $equipment_type = EquipmentType::all()
-            ->pluck('name')
+            ->pluck('id', 'name')
             ->toArray();
-        return $equipment_type;
+        var_dump($equipment_type);
```

Comment:
This was from daebrion's function, and I was tinkering with it but changed it back to leave it how it was.